src/AutorestSwift/Models/Complex/CodeModel.swift

```diff
@@ -60,6 +60,14 @@ class CodeModel: Codable, LanguageShortcut {
         } else if let constantSchema = globalParameter(for: "x-ms-version")?.schema as? ConstantSchema {
             return constantSchema.value.value
         } else {
+            // If there is not `api-version` global parameter is defined, look for apiVersions in Operation for api version
+            for operationGroup in operationGroups {
+                for operation in operationGroup.operations {
+                    if let apiVersions = operation.apiVersions {
+                        return apiVersions[0].version
+                    }
+                }
+            }
```

Comment:
This collection defines, I believe, the apiVersion(s) that an operation supports, which is not the same as getting the api version for the Swagger. Where does the Chat swagger define its api version?

Reply:
Revert the change here.